src/libs/actions/PaymentMethods.js

```diff
@@ -27,4 +31,31 @@ function getPaymentMethods() {
         });
 }
 
-export default getPaymentMethods;
+/**
+ * Calls the API to add a new card.
+ *
+ * @param {Object} params
+ */
+function addBillingCard(params) {
+    API.AddBillingCard({
+        cardNumber: params.cardNumber,
+        cardYear: params.expirationDate.substr(3, 4),
```

Comment:
Isn't the regex for the year expecting just `MM/YY`? this is grabbing 4 digits 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/4073354/130695273-7c53e207-12d7-4e1f-996c-5e95d2f80840.png">

Either way you could probably simplify this to just grab everything from the 3 digit and on with:
```suggestion
        cardYear: params.expirationDate.substr(3),
```